numpy/lib/tests/test_histograms.py

```diff
@@ -397,6 +397,14 @@ def test_histogram_bin_edges(self):
         edges = histogram_bin_edges(arr, bins='auto', range=(0, 1))
         assert_array_equal(edges, e)
 
+    def test_big_arrays(self):
```

Comment:
Thanks.  We need to mark this for large memory use, have a look for `@requires_memory`.  I suspect it should also be marked with `@pytest.mark.slow` to skip on most runs.